### PR TITLE
Add Polish Translation

### PR DESCRIPTION
**Pull request by skerretoo**
Each lesson is translated, if anything was not - please point out what and it'll be rapidly fixed.

### CODE REVIEW

File: _assets/js/lesson_plans.js [hunk 1]
```diff
@@ -6,104 +6,104 @@
   root.lessons = {
     1: {
       0: {
-        startingText: "Writing in Markdown is not that hard!",
```
Comment (gjtorikian) [inline _assets/js/lesson_plans.js:9]: Ah, these changes need to be made in a new `lessons/_es/lesson_plans.js` location.

**Comment by gjtorikian** [comment]
One change required -- the JS content needs to be moved into its own file. Otherwise, looks great. Thank you!

File: _assets/js/lesson_plans.js (hunk 1 above)
Comment (skerretoo) [reply _assets/js/lesson_plans.js]: Should be fixed now ☺️

**Comment by gjtorikian** [comment]
Thank you! Changes should be live shortly.

**Comment by skerretoo** [comment]
It seems the checks have failed, the same for the previous merge. Are you aware of that?

**Comment by gjtorikian** [comment]
I didn't know! Thanks, should be fixed now.

**Pull request merged**